Extension/src/LanguageServer/client.ts

```diff
@@ -3504,12 +3507,32 @@ export class DefaultClient implements Client {
                     character: range.end.character,
                     line: range.end.line
                 }
-            }
+            },
+            copyToClipboard: false
         };
 
+        if (copy) {
+            params.copyToClipboard = true;
+        }
+
         const result: CreateDeclarationOrDefinitionResult = await this.languageClient.sendRequest(CreateDeclarationOrDefinitionRequest, params);
+
+        if (result.edit === undefined) {
+            return;
+        }
         // TODO: return specific errors info in result.
```

Comment:
the comment "// TODO: return specific errors info in result." can be removed since the errors are sent separately from the language server.